README.md

```diff
@@ -644,6 +644,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [GitGuardian](https://www.gitguardian.com) — Keep secrets out of your source code with automated secrets detection and remediation. Scan your git repos for 350+ types of secrets and sensitive files – Free for individuals and teams of 25 developers or less.
   * [Have I been pwned?](https://haveibeenpwned.com) — REST API for fetching the information on the breaches.
   * [hostedscan.com](https://hostedscan.com) — Online vulnerability scanner for web applications, servers, and networks. Ten free scans per month.
+  * [hudsonrock.com](https://cavalier.hudsonrock.com/docs) — Free cybercrime intelligence API calls (free tier) using a free key, supporting use cases such as End User Protection, External Attack Surface Discovery, Assets Intelligence and Domain Intelligence.
```

Comment:
Please link to a product page that details features and pricing, where the free tier needs to be covered.

Reply:
Please check the "Help" section in the link attached (https://cavalier.hudsonrock.com/docs) to view pricing.

Reply:
This does not give details of what can be done with free api keys - limits etc.

Reply:
Each endpoint in this page has a detailed limits and features.

Reply:
I am not studying every end point of your entry to find out what is offered, what is free and so forth.

There are 1000+ entries here over 2000 PRs, I cannot realistically do that.

Your service requires a clear place to look out for the information allowing not just us but also end users to evaluate what is offered

Reply:
All endpoints are free with limited viewing, each endpoint has the explanation of what it does. rate limiting is under "Getting Started".

Reply:
And where to get a free api key? Documentation states to use your own if not testing?

Sorry, I just dont have time to get this information slowly and piece by piece.

The PR template states ``` Pricing information is clearly visible without signup or phone calls``` which this is not.

Feel free to open again once you create a pricing page somewhere with accessible details.